thunx-predicates-querydsl/src/main/java/com/contentgrid/thunx/predicates/querydsl/QueryDslConverter.java

```diff
@@ -38,33 +38,52 @@ public Expression<?> visit(FunctionExpression<?> function) {
 
         // convert all the terms
         var terms = function.getTerms().stream()
-                .map(term -> term.accept(this));
+                .map(term -> term.accept(this))
+                .collect(Collectors.toList());
 
         switch (function.getOperator()) {
             case EQUALS:
```

Comment:
Is this missing a case for `NOT`?

Reply:
I'll add the `NOT` operator

The numeric operators are still missing too - do we really need those ? :grimacing: